clash-prelude/src/Clash/Annotations/Primitive.hs

```diff
@@ -12,7 +12,7 @@ templates.
 {-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE TemplateHaskellQuotes #-}
 
-{-# LANGUAGE Safe #-}
+{-# LANGUAGE Trustworthy #-}
```

Comment:
Why isn't this safe anymore?

Reply:
```
src/Clash/Annotations/Primitive.hs:35:1: error:
    Language.Haskell.TH.Lift: Can't be safely imported!
    The module itself isn't safe.
```

Reply:
But importing `Lift` from the (Trustworthy) [source](https://hackage.haskell.org/package/template-haskell-2.16.0.0/docs/Language-Haskell-TH-Syntax.html) does allow Primitives to be Safe :tada: